### PR TITLE
Initial release

### PR DESCRIPTION
**Pull request by eze-godoy**
## What?

A terraform module to send notifications to any chat application. Currently supports:

- Slack
- Google Chat

## How it works?

using the `chatops_app` variable, the module will use the corresponding script

```hcl
locals {
  ...
  lambda_handler = try(split(".", basename(var.lambda_source_path))[0], "notify_${var.chatops_app}")
  ...
}

module "lambda" {
  source  = "terraform-aws-modules/lambda/aws"

  ...
  handler                        = "${local.lambda_handler}.lambda_handler"
  source_path                    = var.lambda_source_path != null ? "${path.root}/${var.lambda_source_path}" : "${path.module}/functions/notify_${var.chatops_app}.py"
 
  ...
}

```

### CODE REVIEW

**Review by exequielrafaela — APPROVED** [review]

File: versions.tf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.0"
```
Comment (martingaleano) [inline versions.tf]: We try to use `~>` instead of `>=`

File: versions.tf [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 4.8"
```
Comment (martingaleano) [inline versions.tf]: Same here

**Review by martingaleano — APPROVED** [review]

File: versions.tf (hunk 1 above)
Comment (eze-godoy) [reply versions.tf]: Good catch

File: versions.tf (hunk 2 above)
Comment (eze-godoy) [reply versions.tf]: 🫡

**Review by exequielrafaela — APPROVED** [review]

**Pull request merged**